go/fn/runnerProcessor.go

```diff
@@ -38,12 +36,11 @@ func (r *runnerProcessor) config(ctx *Context, o *KubeObject) {
 	case o.IsEmpty():
 		ctx.Result("`FunctionConfig` is not given", Info)
 	case o.IsGVK("v1", "ConfigMap"):
-		var cm corev1.ConfigMap
```

Comment:
If we use ConfigMap a lot in the `fn` package, an alternative is that we define our own type of `ConfigMap` which is a subset of `corev1.ConfigMap`, since we only care a few fields: data, name, namespace, annotations, labels.
It's your call if you want to do this or not :)

Reply:
ConfigMap itself has pretty simple structure. I'd prefer importing as less libraries as possible, especially if it requires additional care for version conflict.